src/pages/fileCentricCart/cartController.js

```diff
@@ -10,14 +11,15 @@ const cartController = () => {
   const ids = cart.fileIds ? cart.fileIds : [];
   const defaultSortDirection = cart.sortDirection === '' ? table.defaultSortDirection || 'asc' : cart.sortDirection;
   const CART_QUERY = defaultSortDirection === 'desc' ? GET_MY_CART_DATA_QUERY_DESC : GET_MY_CART_DATA_QUERY;
+  const defaultSortColumnValue = cart.sortColumn === '' ? table.defaultSortField || '' : cart.sortColumn;
```

Comment:
Are we using this anywhere?

Reply:
yes, otherwise the sort column arrow will be stuck at the default value that you set in the local file.